### DISCUSSION TITLE
v3.7.0

### DISCUSSION THREAD

**Discussion by pp-mo** [SciTools/iris#5452] [Releases]
This is the **minor**  Iris release `v3.7.0`.

Please see the "[What's New](https://scitools-iris.readthedocs.io/en/v3.7.0/whatsnew/index.html)" for further details.

Available from [conda-forge](https://anaconda.org/conda-forge/iris?version=3.7.0&channel=main):
```shell
> mamba install -c conda-forge iris=3.7.0
```

and [PyPI](https://pypi.org/project/scitools-iris/3.7.0):
``` shell
> pip install scitools-iris==3.7.0
```

| Release Feedback |
| :---: |
| [![GH Discussions](https://img.shields.io/badge/github-v3.6.x%20discussion%20%F0%9F%92%AC-yellow?logo=github&logoColor=lightgrey)](https://github.com/SciTools/iris/discussions/5106) |

<hr /><em>This discussion was created from the release <a href='https://github.com/SciTools/iris/releases/tag/v3.7.0'>v3.7.0</a>.</em>

**Comment by bjlittle** [comment]
@pp-mo Why is this release information in the `General` category of our GH Discussions?

We have a dedicated [🚩Releases](https://github.com/SciTools/iris/discussions/categories/releases) category and a dedicated thread for each release i.e., this information belongs in [🦊 v3.7.x](https://github.com/SciTools/iris/discussions/5209).

All releases relating to v3.7.x i.e., release candidate, minor and patch, all belong there.

Why have you chosen to do this differently?

**Comment by pp-mo** [comment]
> Why have you chosen to do this differently?

Ok that was a plain mistake.
Now fixed
